tests/helpers_test.go

```diff
@@ -344,3 +394,55 @@ func createVote(c *qt.C, pid *types.ProcessID, bm *types.BallotMode, encKey *typ
 		Signature:        signature.Bytes(),
 	}
 }
+
+func checkProcessedVotes(t *testing.T, cli *client.HTTPclient, pid *types.ProcessID, voteIDs []types.HexBytes) bool {
+	c := qt.New(t)
+	// Check vote status and return whether all votes are processed
+	txt := strings.Builder{}
+	txt.WriteString("Vote status: ")
+	allProcessed := true
+
+	// Check status for each vote
+	for i, voteID := range voteIDs {
+		// Construct the status endpoint URL
+		statusEndpoint := api.EndpointWithParam(
+			api.EndpointWithParam(api.VoteStatusEndpoint,
+				api.VoteStatusProcessIDParam, pid.String()),
+			api.VoteStatusVoteIDParam, voteID.String())
+
+		// Make the request to get the vote status
+		body, statusCode, err := cli.Request("GET", nil, nil, statusEndpoint)
+		c.Assert(err, qt.IsNil)
+		c.Assert(statusCode, qt.Equals, 200)
+
+		// Parse the response body to get the status
+		var statusResponse api.VoteStatusResponse
+		err = json.NewDecoder(bytes.NewReader(body)).Decode(&statusResponse)
+		c.Assert(err, qt.IsNil)
+
+		// Verify the status is valid
+		c.Assert(statusResponse.Status, qt.Not(qt.Equals), "")
+
+		// Check if the vote is processed
+		if statusResponse.Status != "processed" {
```

Comment:
do we have any constants for "processed" ? any problems with mixedcase response status string ?